### PR TITLE
modify qiniu code adding upload large file

### PR DESCRIPTION
**Pull request by m20082008m**
from mayanpeng
the code just submitted have a bit questions.

### CODE REVIEW

File: backend/drivers/qiniu/qiniu.go [hunk 1]
```diff
@@ -27,17 +29,55 @@ func qiniusave(file string) (url string, err error) {
...
+	}
+	defer fil.Close()
+	fi, err := fil.Stat()
+	if err != nil {
+		return "", err
+	}
+	fsize := fi.Size()
 
-	url = "http://" + setting.Endpoint + "/" + key
+	//conf.ACCESS_KEYdd = setting.AccessKeyID
```
Comment (MabinGo) [inline backend/drivers/qiniu/qiniu.go:43]: please delete the redundant codes

File: backend/drivers/qiniu/qiniu.go.bak [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package qiniu
```
Comment (MabinGo) [inline backend/drivers/qiniu/qiniu.go.bak:1]: .bak file must be deleted

File: backend/drivers/qiniu/qiniu_test.go [hunk 3]
```diff
@@ -15,11 +16,14 @@ func Test_qiniusave(t *testing.T) {
 		t.Error(err)
 	}
 
+	//file := "/home/ma/share/VMware.exe"
```
Comment (MabinGo) [inline backend/drivers/qiniu/qiniu_test.go:19]: please delete the redundant codes

**Comment by m20082008m** [comment]
the code is old, I update in new request

**Pull request closed without merging**